src/assets/styles/fonts.scss

```diff
@@ -0,0 +1,69 @@
+/*
+** Fonts
+*/
+$fonturl: '../../assets/fonts';
```

Comment:
Thinking about it more, I think the font files should be in some `./public` or `./static` directory or something like that. Is that too hard to pull off? If not, can that be done? 😀

Reply:
Moved to fonts to `./public/fonts` folder